core/src/main/java/org/apache/spark/SparkThrowable.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark;
+
+import org.apache.spark.annotation.Experimental;
+
+/**
+ * :: Experimental ::
+ *
+ * Interface mixed into Throwables thrown from Spark.
+ *
+ * - For backwards compatibility, existing Throwable types can be thrown with an arbitrary error
+ *   message with a null error class. See [[SparkException]].
+ * - To promote standardization, Throwables should be thrown with an error class and message
+ *   parameters to construct an error message with SparkThrowableHelper.getMessage(). New Throwable
+ *   types should not accept arbitrary error messages. See [[SparkArithmeticException]].
+ */
+@Experimental
+public interface SparkThrowable {
+    // Succinct, human-readable, unique, and consistent representation of the error category
```

Comment:
Let's keep them 2-spaced

Reply:
I know many of other java files are not but strictly it should be 2-spaced

Reply:
Done (I think) - can you confirm?